api/src/services/specifications.test.ts

```diff
@@ -122,17 +124,17 @@ describe('Integration Tests', () => {
 							schema: {
 								name: 'test_table',
 							},
-						])
-					);
-					vi.spyOn(RelationsService.prototype, 'readAll').mockImplementation(vi.fn().mockReturnValue([]));
-
-					const spec = await service.oas.generate();
-					expect(spec.components?.schemas).toEqual({
-						ItemsTestTable: {
-							type: 'object',
-							properties: {
-								id: {
-									nullable: false,
+						};
+
+						jest
+							.spyOn(CollectionsService.prototype, 'readByQuery')
+							.mockImplementation(jest.fn().mockReturnValue([collection]));
+
+						jest.spyOn(FieldsService.prototype, 'readAll').mockImplementation(
+							jest.fn().mockReturnValue([
+								{
+									collection: collection.collection,
+									field: 'id',
```

Comment:
Picked wrong side while merging? 😸

Reply:
🤫

Reply:
All joking aside, no, it was easier to merge it "wrong" to make sure the test was fully up to date with main, and then rework jest->vi after the fact (see commit below :) )

Reply:
I see 😃 I'll be quiet and let you do your work 🤫